docs/1.2/pbft/architecture.md

```diff
@@ -239,210 +222,191 @@ message PbftSeal {
 }
 ```
 
-## Message Types {#pbft-arch-message-types}
+### Message Types {#pbft-arch-message-types}
 
 A Sawtooth PBFT message has one of the following types:
 
--   `PrePrepare`: Sent by the primary node after it has published a new
-    block
--   `Prepare`: Broadcast by every node in the `Preparing` phase
--   `Commit`: Broadcast by every node in the `Committing` phase
--   `ViewChange`: Sent by any node that suspects that the primary is
-    faulty
--   `NewView`: Sent by the node that will be the new primary to complete
-    a view change
--   `Seal`: Proves that a block was committed after `2f + 1` nodes
-    agreed to commit it
--   `SealRequest`: Sent by a node that is requesting a consensus seal
-    for the block that was committed at a given sequence number
-
-# PBFT Operation {#pbft-operation-label}
-
-The Sawtooth PBFT algorithm starts with initialization, then operates in
-one of two modes:
-
--   `Normal mode <normal-mode-label>`{.interpreted-text role="ref"} for
-    processing blocks
--   `View Changing mode <view-changing-mode-label>`{.interpreted-text
-    role="ref"} for switching to a different primary node
-
-::: note
-::: title
-Note
-:::
-
-The original PBFT definition includes a checkpointing procedure that is
-responsible for garbage collection of the log. Sawtooth PBFT does not
-implement this checkpointing procedure; instead, it cleans the log
-periodically during its normal operation. For more information, see
-`log-pruning-label`{.interpreted-text role="ref"}.
-:::
-
-## Initialization
+- `PrePrepare`: Sent by the primary node after it has published a new block
+- `Prepare`: Broadcast by every node in the `Preparing` phase
+- `Commit`: Broadcast by every node in the `Committing` phase
+- `ViewChange`: Sent by any node that suspects that the primary is faulty
+- `NewView`: Sent by the node that will be the new primary to complete a view
+  change
+- `Seal`: Proves that a block was committed after `2f + 1` nodes agreed to
+  commit it
+- `SealRequest`: Sent by a node that is requesting a consensus seal for the
+  block that was committed at a given sequence number
+
+## PBFT Operation {#pbft-operation-label}
+
+The Sawtooth PBFT algorithm starts with initialization, then operates in one of
+two modes:
+
+- [Normal mode](#normal-mode-label) for processing blocks
+- [View Changing mode](#view-changing-mode-label) for switching to a different
+  primary node
+
+> **Note**
+>
+> The original PBFT definition includes a checkpointing procedure that is
+> responsible for garbage collection of the log. Sawtooth PBFT does not
+> implement this checkpointing procedure; instead, it cleans the log
+> periodically during its normal operation. For more information, see
+> `log-pruning-label`{.interpreted-text role="ref"}.
```

Comment:
Should interpreted-text be removed here?

Reply:
Fixed